db/schema.rb

```diff
@@ -45,7 +44,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.text "accepted_file_type", default: ".pdf"
-    t.date "deletion_date", default: "2020-10-15", null: false
```

Comment:
I think this is the result of 
```
class AddDeletionDateToAssignments < ActiveRecord::Migration[6.1]
  def up
    add_column :assignments,
               :deletion_date,
               :date, null: false, default: (Term.active&.end_date || (Date.today + 180.days)) + 15.days
```
in 20220125162730_add_deletion_date_to_assignments.rb which is a little unfortunate because this make sthe default value dependent on when the migration is actually executed.

Reply:
Ok, but why did you change the default deletion date in this PR then? Or do you mean it was somehow automatically changed by Rails? But should we include this change here?

Reply:
This code is exexuted at the time when you run the migration. Therefore, depending on what the active term is in your development(!) environment, Rails generates the date in this migration. Even if the active term in development is set at some fantasy date in the stone age. In particular, this also differs from the setting in production. Fortunately, no harm can be done because of these lines
```
  validates :deletion_date, presence: true
  validate :deletion_date_cannot_be_in_the_past
```
in `app/models/assignment.rb`. However, they also imply that the current deletion_date in the `schema.rb` is invalid. I will add a commit that hard fixes the default deletion date to somewhere in the distant future.